pkg/virt-handler/cache/cache_test.go

```diff
@@ -284,12 +290,16 @@ var _ = Describe("Domain informer", func() {
 
 			domain := api.NewMinimalDomain("test")
 			domainManager.EXPECT().ListAllDomains().Return([]*api.Domain{domain}, nil)
+			domainManager.EXPECT().GetGuestOSInfo().Return(&api.GuestOSInfo{})
+			domainManager.EXPECT().InterfacesStatus(domain.Spec.Devices.Interfaces).Return([]api.InterfaceStatus{})
 			// now prove if we make a change, like adding a label, that the resync
 			// will pick that change up automatically
 			newDomain := domain.DeepCopy()
 			newDomain.ObjectMeta.Labels = make(map[string]string)
 			newDomain.ObjectMeta.Labels["some-label"] = "some-value"
 			domainManager.EXPECT().ListAllDomains().Return([]*api.Domain{newDomain}, nil)
+			domainManager.EXPECT().GetGuestOSInfo().Return(&api.GuestOSInfo{Name: "fedora"})
+			domainManager.EXPECT().InterfacesStatus(newDomain.Spec.Devices.Interfaces).Return([]api.InterfaceStatus{api.InterfaceStatus{Name: "ethx"}})
```

Comment:
Isn't [this](https://github.com/kubevirt/kubevirt/pull/5428/commits/3e9ba98a98acc41134788bb1e85bfca9eeeea9e0#diff-6406f1444186b45ac44ceb8e369fd9b6ce0c9a4cc6f313bf80e815f0f33e705bR327) part missing?

Reply:
Edy asked to remove it
See here please why
https://github.com/kubevirt/kubevirt/pull/5560#discussion_r626322993

Reply:
I see, so please drop the "special" mocking here ("fedora", "ethx"). It is confusing.